app/controllers/concerns/verify_sp_attributes_concern.rb

```diff
@@ -7,6 +7,20 @@ def needs_completions_screen?
         consent_was_revoked?)
   end
 
+  def needs_completion_screen_reason
+    return nil unless sp_session[:issuer].present?
```

Comment:
`unless` messes with my brain, I find this easier to read:

```suggestion
    return nil if sp_session[:issuer].blank?
```